ecosystem/sep-0024.md

```diff
@@ -697,6 +696,18 @@ For example:
 }
 ```
 
+## Changes from SEP-6
+
+There is a small set of changes when upgrading from SEP-6 to SEP-24.
+
+1. SEP-24 now requires authentication on all endpoints.  The `authentication_required` flag is removed from the `/info` endpoint since authentication is assumed.
+1. `GET /deposit` and `GET /withdraw` have been replaced with [`POST /transactions/deposit/interactive`](#deposit) and [`POST /transactions/withdraw/interactive`](#withdraw).  This is for security purposes to keep submitted personally identifiable information out of URL query parameters.
+1. Remove the `type` parameter from the `/deposit` and `/withdraw` endpoints as this is information collected in the interactive flow.
+1. Removed `external_extra` transaction property since this should all live in a human readable ` more_info_url`.
+1. Changed the response of the deposit and withdraw endpoints from 403 to 200 since this is the expected flow.
```

Comment:
I don't remember ordinary deposit responses being a 403 response, do you mean specifically error responses?

Reply:
In SEP-6 when you post to /deposit, you expect a 403 response with the URL for the interactive flow https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#3-customer-information-needed-interactive